cumulus_library_opioid/vocab/template_sql/create_excluded_rela_types.sql.jinja

```diff
@@ -0,0 +1,12 @@
+CREATE VIEW {{ prefix }}{{ steward }}_excluded_{{ name }} AS 
+SELECT 
+    rxcui1,
+    tty1,
+    rel,
+    rela,
+    rxcui2,
+    tty2,
+    str2
```

Comment:
Just as an aside, these field names are all meaningless to someone not steeped in this repo. I know some of this is inherited. But phew.

Reply:
i :hate: these names. but it's either exactly an NIH label, or an NIH label with a number, so it makes it easier to look up in their documentation. 

it's certainly an artifact of a bygone age.